### DISCUSSION TITLE
cmd message says whisper-faster is not compatible with the version of Windows

### DISCUSSION THREAD

**Discussion by limheat** [Purfview/whisper-standalone-win#275] [Q&A] [answered]
Asking for someone else.

Running whisper-faster (r160.10) on 32-bit system.

Got this in cmd: "This verison of this file is not compatible with the version of Windows you're running. Check your computer's system information to see whether you need an x86 (32-bit) or x64 (64-bit) version of the program"
Screenshot:
![36f825c90eccfe1be2b0e5f8c25c52381849141391](https://github.com/Purfview/whisper-standalone-win/assets/145899967/5ca1ed3e-6899-42ff-bdbc-6afa14ae5458)
the message says "This verison of [whisper-faster] is not compatible with the version of Windows you're running..."

According to the readme info, whisper faster is compatible with 32-bit win 7 systems.

Is there some way to fix this problem?

**Comment by Purfview** [comment, marked as answer]
It's not compatible with 32-bit.

Btw, r160.10 is some ancient version.
